src/app/containers/SocialEmbed/__snapshots__/index.test.jsx.snap

```diff
@@ -142,9 +142,9 @@ exports[`SocialEmbedContainer AMP should render correctly 1`] = `
           >
             <a
               class="c3"
-              href="#skip-twitter-content-c063a42a"
+              href="#skip-twitter-content-2"
             >
-              Skip Twitter content
+              Skip Twitter content, 2
```

Comment:
Yes, this is an accessibility requirement. All links should be unique, otherwise users may not be able to orientate themselves. Initially this was going to be a username, but we do not always have access to this information, so it was decided we'd use an index.